File: annotationprocessortesting/src/main/java/com/tickaroo/tikxml/annotationprocessing/namespace/RootDefaultNamespace.java
package com.tickaroo.tikxml.annotationprocessing.namespace;

import com.tickaroo.tikxml.annotation.Element;
import com.tickaroo.tikxml.annotation.PropertyElement;
import com.tickaroo.tikxml.annotation.Xml;

/**
 * @author Mariusz Saramak
 */
@Xml(writeNamespaces = { "http://www.w3.org/1998/Math/MathML" })
public class RootDefaultNamespace {
  @PropertyElement(name = "name") String name;
  @Element(name = "child") Child child;
}

File: annotationprocessingtesting-kt/src/main/java/com/tickaroo/tikxml/annotationprocessing/namespace/RootDefaultNamespace.kt
package com.tickaroo.tikxml.annotationprocessing.namespace

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

/**
 * @author Mariusz Saramak
 */
@Xml(writeNamespaces = ["http://www.w3.org/1998/Math/MathML"])
class RootDefaultNamespace {
    @PropertyElement(name = "name")
    var name: String? = null
    @Element(name = "child")
    var child: Child? = null
}
